unified-runtime/scripts/core/manifests.yml

```diff
@@ -8,7 +8,7 @@
 # See YaML.md for syntax definition
 #
 # NOTE: device_types must be explictly listed. Refrain from using 
-#       $X_DEVICE_TYPE_ALL. See https://github.com/intel/llvm/issues/17527.
```

Comment:
I think we can consider this issue addressed with the change here, can you put a "Fixes" clause in the PR description so it gets closed automatically

Reply:
Done.